support/builder/config.sh

```diff
@@ -42,7 +32,7 @@ web_url = "$GITHUB_WEB_URL"
 client_id = "$GITHUB_CLIENT_ID"
 client_secret = "$GITHUB_CLIENT_SECRET"
 app_id = $GITHUB_APP_ID
-app_url = "${GITHUB_WEB_URL}/apps/$(echo ${GITHUB_APP_NAME} | tr "[:upper:]" "[:lower:]" | tr " " "-")"
+app_url = "${GITHUB_APP_NAME}"
```

Comment:
I'm not sure this'll work, since the UI wants the full URL and not just the name. If I'm reading this right, we should should probably rename this to `GITHUB_APP_URL` and then update the sample config to use the URL to the dev-studio app as well. Does that sound right?

Reply:
Yeah that makes sense. I updated the value of the variable logically but not semantically. Got it patched up.